src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -412,6 +412,7 @@
     <OutOfBoxFramework Include="netcore50">
       <RuntimeId>win10</RuntimeId>
     </OutOfBoxFramework>
+    <OutOfBoxFramework Condition="'$(IsUap101Defined)'=='true'" Include="uap10.1"/>
```

Comment:
Hmm. That's also possible. I don't like splitting OutOfBoxFramework items across multiple files, but I admit that it'll make like much easier.